src/plots/cartesian/axis_defaults.js

```diff
@@ -44,6 +44,10 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
             (options.title || (letter.toUpperCase() + ' axis')) +
             ' title';
 
+    function coerce2(attr, dflt) {
+        return Lib.coerce2(containerIn, containerOut, layoutAttributes, attr, dflt);
```

Comment:
non- :no_entry_sign: but we should make `coerce2` use the same pattern as `coerceFont` where a shortcut regular `coerce` function is passed in.